node_modules/col-activities/lib/api.js

```diff
@@ -481,7 +481,8 @@ var getActivitiesForAssetId = module.exports.getActivitiesForAssetId = function(
  * @param  {Activity}         callback.activity                   The created activity
  */
 var createActivity = module.exports.createActivity = function(course, user, type, objectId, objectType, metadata, actor, callback) {
-  metadata = metadata || {};
+  // Copy metadata object so that modifications don't affect any other activities that may use the same metadata.
+  metadata = _.extend({}, metadata);
```

Comment:
@paulkerschen do we need to `cloneDeep` metadata prior to extend?  Similar to PR #558